core/tests/protractor_desktop/navigation.js

```diff
@@ -139,6 +139,15 @@ describe('Meta Tags', function() {
   });
 });
 
+describe('DEV MODE Test', function() {
+  it('should not show Dev Mode label', function() {
```

Comment:
Hmmm... Shouldn't this be only in production mode? If I run this without the prod_env flag this test will fail?

Reply:
Yes.
I've refactored it. PTAL.
Thanks.